### DISCUSSION TITLE
How to run task every N seconds?

### DISCUSSION THREAD

**Discussion by ukolovda** [cybertec-postgresql/pg_timetable#308] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Can I run task, for example, every 10 seconds? In my project I need push SMS or e-mail from queue table, and I want run it faster then every minute.

**Describe the solution you'd like**
It can be extended time format with additional parameter at start, i.e. 
`*/10s * * * * *`
or other

**Describe alternatives you've considered**
I can make cycle in code with sleep between running main task.

**Additional context**

**Comment by pashagolub** [comment, marked as answer]
Hello, you can try interval chains (jobs), e.g. `@every 10 seconds`. Or you may consider using `@after 10 seconds` if you want to wait for your previous job to be finished.
